dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -246,6 +248,35 @@ def test_add_profilers(self, *mocks):
         self.assertEqual("multiple files", merged_profile.encoding)
         self.assertEqual("multiple files", merged_profile.file_type)
 
+    def test_add_profiles_null_count_not_enabled(self):
+        profiler1_options_null_count = ProfilerOptions()
+        profiler1_options_null_count.set(
+            {"row_statistics.null_count.is_enabled": False}
+        )
+        profiler2_options_null_count = ProfilerOptions()
+        profiler2_options_null_count.set({"row_statistics.null_count.is_enabled": True})
+
+        data = pd.DataFrame([1, None, 3, 4, 5, None, 1])
+        with test_utils.mock_timeit():
+            profile1 = dp.StructuredProfiler(
+                data[:2], options=profiler1_options_null_count
+            )
+            profile2 = dp.StructuredProfiler(
+                data[2:], options=profiler2_options_null_count
+            )
+
+        self.assertEqual(0, profile1.row_has_null_count)
+        self.assertEqual(0, profile1.row_is_null_count)
+        self.assertLess(0, profile2.row_has_null_count)
+        self.assertLess(0, profile2.row_is_null_count)
```

Comment:
This is good, but the test itself doesn't describe the intent for this check.
`test_add_profiles_null_count_not_enabled` to me would suggest only validating adding profiles.

moreover, below, it checks for adding profiles that are both enabled and disabled.

Maybe, instead rename to `test_adding_profiles_of_mismatched_null_count_options`
and move the assertEquals above to a different test where we just check the functionality when the stat is disabled.